packages/legend-query-builder/src/stores/fetch-structure/tds/projection/QueryBuilderProjectionStateBuilder.ts

```diff
@@ -96,47 +86,132 @@ export const processTDSProjectExpression = (
     parentLambda,
     queryBuilderState,
   );
+};
+
+const processTDSProjectColExpression = (
```

Comment:
These functions all have similar names and seem to do similar things. Do you think it would be useful to add a comment above each function explaining what each one does and when/why it is used?